docs/src/codegen.md

```diff
@@ -414,6 +414,26 @@ pwsh bin/Debug/netX/playwright.ps1 codegen --load-storage=auth.json github.com/m
 
 <img width="1394" alt="github signed in showing use of load storage scharp" src="https://user-images.githubusercontent.com/13063165/220928354-caa0e958-fe09-4125-9b54-67483064da51.png" />
 
+#### Use existing userDataDir
+
+Run `codegen` with `--user-data-dir` to set a fixed [user data directory](https://playwright.dev/docs/api/class-browsertype#browser-type-launch-persistent-context-option-user-data-dir) for the browser session. If you point this at your existing browser's user data directory, codegen will use your existing browser profile and have access to your authentication state. Remember that any authentication that codegen receives from using your browser configuration must be replicated at test time using fixtures or `before*` hooks, otherwise the UI you're trying to evaluate might not be available when executing the test.
+
+```bash js
+npx playwright codegen --user-data-dir=/path/to/your/browser/data/ github.com/microsoft/playwright
```

Comment:
This won't work for a user who has a different profile than `'Default'`. The story is a bit more complicated there that we should teach them how to get this path, what is `'Default'` etc. I wonder if we want to enter this territory or if we should rely on third-party blog posts like [this](https://dev.to/mxschmitt/running-playwright-codegen-with-existing-chromium-profiles-5g7k) one. The main customers of this are Chromium anyways and doing this with `page.pause()` seems to have worked fine in the past.